pkg/aws/ec2/mock/mock.go

```diff
@@ -357,6 +387,113 @@ func (e *API) UnassignPrivateIpAddresses(ctx context.Context, eniID string, addr
 	return fmt.Errorf("Unable to find ENI with ID %s", eniID)
 }
 
+func assignPrefixToEni(e *API, eni *eniTypes.ENI, prefixes int32) error {
```

Comment:
Nit: Noticing that a lot of references to ENI are all capitalized so let's be consistent with using `ENI`. If they are all lowercased already then we can leave it so.

Reply:
Refactored to `assignPrefixToENI()`